### DISCUSSION TITLE
Deployer 7 starts to count from 1 again

### DISCUSSION THREAD

**Discussion by jasperf** [deployphp/deployer#2713] [Help needed]
#### Deployer resets counting

On deploying with new Deployer 7 (was using Deployer 6 before)  I realized Deployer starts counting from 1 again. If you would do ten releases and you had 5 with the older Deployer 6 this would clash. This as you would already have release 1-5. 

#### Rollbackup Issues
Other issue is that if I would like to do a restore this would probably not work as Deployer started counting from the beginning again so a release 626, which we have on staging would not be used for a rollback or would it?

Guess this cannot be helped?

**Comment by antonmedv** [comment]
Yes, releases history not compatible between v6 and v7. 

But some migration code can be implemented. I’ll take a look at this. 

To release after 626 use

    dep deploy -o release_name=627

**Reply by antonmedv** [reply]
I think better migration docs are needed. Will work on it. No further code changes are needed. 

Let’s push v7 release forward!

**Reply by jasperf** [reply]
We can add a file to the pull request I added where I am already adding documentation.

**Reply by jasperf** [reply]
Will probably delete the old Deployer 6 releases in  a bit. This to make sure a new release does not clash with one of the older ones.

**Reply by antonmedv** [reply]
I wrote a detailed guide here https://deployer.org/docs/7.x/UPGRADE

**Reply by jasperf** [reply]
> I wrote a detailed guide here https://deployer.org/docs/7.x/UPGRADE

Thanks.  Perhaps only need a note that post upgrade you start with release 1 from scratch

**Reply by antonmedv** [reply]
Nope if dep deploy -o release_name=43
